### DISCUSSION TITLE
How do I import My Clippings?

### DISCUSSION THREAD

**Discussion by ugurtarlig** [hadynz/obsidian-kindle-plugin#239] [Q&A]
First off: Thank you very much for this plugin! 

I am trying to find out how to import "My Clippings.txt", but I haven't found any info in the README, nor in the code.

Any help for how to import My Clippings file is very much appreciated.

**Comment by joshdavislight** [comment]
Once you have installed the plugin, there is a button on the left hand toolbar that says "Sync Your Kindle Highlights." Once you click that, you can upload your clippings.txt file.
